.github/workflows/build.yml

```diff
@@ -19,165 +26,77 @@ env:
-          platform="${{ matrix.platform }}"
-          bash ./codecov -F "${platform%%-*}"
-      - name: Generate coverage HTML report
-        run: go tool cover -html=coverage.txt -o coverage.html
-      - name: Upload coverage report
-        uses: actions/upload-artifact@v2
-        with:
-          name: test-coverage-report-${{ matrix.platform }}
-          path: coverage.html
-
-  configure:
-    runs-on: ubuntu-latest
-    outputs:
-      version: ${{ steps.get_version.outputs.version }}
-    steps:
-      - name: Get version
-        id: get_version
-        run: |
-          VERSION="${GITHUB_REF##*/}"
-          echo "VERSION=${VERSION}"
-          echo "::set-output name=version::${VERSION}"
+          echo "GO_VERSION=${GO_VERSION}"
+          echo "::set-output name=go_version::${GO_VERSION}"
 
   build:
     runs-on: ubuntu-latest
-    needs: [configure, test-current-cov]
-    if: startsWith(github.ref, 'refs/tags/v')
+    needs: [configure]
     env:
-      VERSION: ${{ needs.configure.outputs.version }}
+      VERSION: ${{ needs.configure.outputs.k6_version }}
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
       - name: Install Go
         uses: actions/setup-go@v2
         with:
-          # We plan to move to 1.18.x after the release of the first patch.
-          go-version: 1.18.x
-      - name: Install package builders
-        env:
-          GO111MODULE: 'off'
+          go-version: ${{ needs.configure.outputs.go_version }}
+      - name: Install nfpm (dep and rpm package builder)
         run: |
-          gopath="$(go env GOPATH)"
-          go get github.com/Masterminds/glide
-          go get -d github.com/mh-cbon/go-bin-deb \
-              && (cd "$gopath/src/github.com/mh-cbon/go-bin-deb" \
-              && glide install \
-              && go install)
-          go get -d github.com/mh-cbon/go-bin-rpm \
-              && (cd "$gopath/src/github.com/mh-cbon/go-bin-rpm" \
-              && glide install \
-              && go install)
-          sudo apt-get update -y
-          sudo apt-get install -y fakeroot rpm
-      - name: Generate Windows binary metadata
+          go install github.com/goreleaser/nfpm/v2/cmd/nfpm@v2.16.0
+      - name: Install goversioninfo (.syso file creator)
         run: |
-          (cd && GO111MODULE=on go get github.com/josephspurrier/goversioninfo/cmd/goversioninfo@v1.2.0)
-          IFS=. read -a version <<< "$(echo $VERSION | sed 's:[^0-9\.]::g')"
+          go install github.com/josephspurrier/goversioninfo/cmd/goversioninfo@v1.4.0
+      - name: Generate Windows binary metadata (.syso files)
+        run: |
+          IFS=. read -a version_parts <<< "${VERSION#v}"
+          IFS=- read -a version_patch <<< "${version_parts[2]}"
+
           # Need a blank versioninfo.json for the CLI overrides to work.
           echo '{}' > versioninfo.json
+          set -x
```

Comment:
Again - maybe a comment why this is done

Reply:
Resolved by the [latest force push](https://github.com/grafana/k6/compare/7bf212fa06fa9dbeb4f7ce7baee935445e1b878c..ae28e070b23f315cab179da884814f2a7ee22d18)